piezo_web_app/PiezoWebApp/src/models/validation_rule.py

```diff
@@ -15,12 +15,20 @@ def __init__(self, rule_dict):
         self._maximum = self._safe_get_value('maximum', rule_dict)
         self._options = self._safe_get_value('options', rule_dict)
 
+        self._type = self._safe_get_type('type', rule_dict)
+
     @staticmethod
     def _safe_get_value(key, dictionary):
         if key in dictionary:
             return dictionary[key]
         return None
 
+    @staticmethod
+    def _safe_get_type(key, dictionary):
+        if key in dictionary:
+            return dictionary[key]
+        return "string"
```

Comment:
It would be simpler here to extend the `_safe_get_value` method. Something like:
``` python
@staticmethod
    def _safe_get_value(key, dictionary, value_if_missing=None):
        if key in dictionary:
            return dictionary[key]
        return value_if_missing
```

This could then be called above as:
``` python
self._type = self._safe_get_value('type', rule_dict, value_if_missing="string")
```